From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Tom Levy <tomlevy93@gmail.com>
Date: Mon, 22 Jun 2020 07:58:59 +0000
Subject: cherry-pick: Fix example judgehost username in manual

The default username created by etc/genrestapicredentials is
"judgehost" (singular), not "judgehosts" (plural).

(cherry picked from commit b3a7ac0a27d75f4051f9f9f108a09afe8b3168c4)
---
 doc/manual/install-judgehost.rst | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/doc/manual/install-judgehost.rst b/doc/manual/install-judgehost.rst
index 31a26bb2f..7dbab777e 100644
--- a/doc/manual/install-judgehost.rst
+++ b/doc/manual/install-judgehost.rst
@@ -152,9 +152,9 @@ On each judgehost, copy from the domserver (or create) a file
 ``etc/restapi.secret`` containing the id, URL,
 username and password whitespace-separated on one line, for example::
 
-  default http://example.edu/domjudge/api/  judgehosts  MzfJYWF5agSlUfmiGEy5mgkfqU
+  default http://example.edu/domjudge/api/  judgehost  MzfJYWF5agSlUfmiGEy5mgkfqU
 
-The password here must be identical to that of the ``judgehosts`` user
+The password here must be identical to that of the ``judgehost`` user
 in the admin web interface. Multiple lines may be specified to allow a
 judgedaemon to work for multiple domservers. The id in the first column
 is used to differentiate between multiple domservers, and should be
-- 
2.20.1

